test/services/hyperlink-utility.test.js

```diff
@@ -139,25 +129,21 @@ describe('supportsHyperlink', () => {
 });
 
 describe('convertToHyperlink', () => {
-  describe('test hyperlink generation for dummyURL and dummyText on macOS', () => {
-    describe('test for iTerm', () => {
+  describe('test for iTerm', () => {
+    describe('test hyperlink generation for dummyURL and dummyText on macOS', () => {
       beforeEach(() => {
+        flush();
         process.env.TERM_PROGRAM = 'iTerm.app';
         process.env.TERM_PROGRAM_VERSION = '3.1.0';
+        process.argv = ['--hyperlink=true'];
       });
       afterEach(() => {
-        flush();
+        process.argv = [];
         process.env = ORIG_ENV;
       });
-      testLink.cmdTestLink([]).it('test', (ctx) => {
+      testLink.cmdTestLink([]).it('verify the contents of dummy Command', (ctx) => {
         const result = convertToHyperlink('MORE INFO', 'https://twilio.com/docs/dummyCmd').isSupported;
-        if ('CI' in process.env) {
```

Comment:
Since the hyperlink flag` --hyperlink` is set to `true`, it'll be honoured in CI environments too.